javascript/javascript-import-and-require.md

```diff
@@ -0,0 +1,79 @@
+# JavaScript - Require & Import
+
+## Project Time
+
+About 45 Minutes
+
+- 30 minutes for the video and reference material
+- 10 minutes for Check for Understanding
+- 5 minutes for verify
+
+## Prerequisites
+
+- JavaScript 7
+- [JavaScript Import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
+- [JavaScript Require](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+
+## Motivation
+
+JavaScript is one of most used language. It is lightweight, Interpreted and Just-in-time compiled programming language with first-class function. While it is most well-known as the scripting language for webpage, Many no-browser environmemts also use it. JavaScript is a prototype-based, multi-paradigm, single-threaded, dynamic language, supporting object-oriented, imperative and declarative style.

```

Comment:
Please use proper grammar: Javascript is one of `the` most used language. Do not use capital letters in between the sentences. Try to add something about import & require here.

Reply:
They are not learning what JS is in this lesson.  They are learning about import and require.
check out the readme to see what else is included in the curriculum. https://github.com/Techtonica/curriculum/blob/master/README.md#week-3---javascript-fundamentals

Reply:
Also, feel free to use other resources, but if you are going to copy text, cite your source and provide a link.